packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,53 @@
       <protected>false</protected>
       <shortDescription>Opens in the browser tab</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
```

Comment:
if the error applies to any component, why the c-quantic-pager in shortDescription?

Reply:
ah that's not the value that will be displayed, the value is in the tag `value`.
in the tag `shortDescription` I'm just adding an example of what the label should look like.

Reply:
ahh ok ok